CONTRIBUTING.md

```diff
@@ -0,0 +1,156 @@
+# Contributing
+
+Contributions are welcome, and they are greatly appreciated! Every little bit helps,
+and credit will always be given.
+
+## Areas of contribution
+
+Contributions are welcome across the entire project:
+
+- Code
+- Documentation
+- Testing
+- Packaging/distribution
+
+## Contributing workflow
+
+### New Contributors
+
+1. Search the [issues](https://github.com/PaloAltoNetworks/terraform-aws-vmseries-modules.git/issues) to see if there is an existing issue. If not, open an issue (note the issue ID).
+
+1. Fork the repository to your personal namespace (only need to do this once).
+
+1. Clone the repo from your personal namespace.
+
+   `git clone https://github.com/{username}/tterraform-aws-vmseries-modules.git`
```

Comment:
There is a type here, `tteraform` with two T's

Reply:
```suggestion
   `git clone https://github.com/{username}/terraform-aws-vmseries-modules.git`
```